src/modules/setup.ts

```diff
@@ -63,6 +72,10 @@ class ReplaceVariablesStream extends Transform {
         this.push(result);
         callback();
     }
+
+    private _formatResultForException(result: string, startingLine: string, noticeComment: string) {
```

Comment:
Right now what this does is to inject the notice after the start line, so lets call it just that.
`_injectNoticeAfterStartLine`